src/main/docker/Dockerfile

```diff
@@ -14,11 +14,129 @@
+
+RUN wget -O python.tar.xz "https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz" && \
+    echo "$PYTHON_SHA512 python.tar.xz" | sha512sum --strict --check - && \
+    mkdir -p /usr/src/python && \
+    tar -xJC /usr/src/python --strip-components=1 -f python.tar.xz && \
+    rm python.tar.xz
+
+RUN cd /usr/src/python && \
+    ./configure \
+        --enable-loadable-sqlite-extensions \
+        --enable-optimizations \
+        --enable-option-checking=fatal \
+        --enable-shared \
+        --with-system-expat \
+        --with-system-ffi \
+        --without-ensurepip && \
+    make -j8 \
+# https://github.com/docker-library/python/issues/160#issuecomment-509426916
+        PROFILE_TASK='-m test.regrtest --pgo \
+            test_array \
+            test_base64 \
+            test_binascii \
+            test_binhex \
+            test_binop \
+            test_bytes \
+            test_c_locale_coercion \
+            test_class \
+            test_cmath \
+            test_codecs \
+            test_compile \
+            test_complex \
+            test_csv \
+            test_decimal \
+            test_dict \
+            test_float \
+            test_fstring \
+            test_hashlib \
+            test_io \
+            test_iter \
+            test_json \
+            test_long \
+            test_math \
+            test_memoryview \
+            test_pickle \
+            test_re \
+            test_set \
+            test_slice \
+            test_struct \
+            test_threading \
+            test_time \
+            test_traceback \
+            test_unicode \
+        ' && \
+    make install && \
+    ldconfig
+
+# Remove test files
+RUN find /usr/local -depth \
+        \( \
+            \( -type d -a \( -name test -o -name tests -o -name idle_test \) \) \
+            -o \
+            \( -type f -a \( -name '*.pyc' -o -name '*.pyo' \) \) \
+        \) -exec rm -rf '{}' + && \
+    python3 --version
+
+# Some useful symlinks
+RUN cd /usr/local/bin && \
+    ln -s idle3 idle && \
+    ln -s pydoc3 pydoc && \
+    ln -s python3 python && \
+    ln -s python3-config python-config
+
+# Install pip
+ENV PYTHON_PIP_VERSION=19.3.1 \
+    PYTHON_GET_PIP_URL=https://github.com/pypa/get-pip/raw/ffe826207a010164265d9cc807978e3604d18ca0/get-pip.py \
+    PYTHON_GET_PIP_SHA256=b86f36cc4345ae87bfd4f10ef6b2dbfa7a872fbff70608a1e43944d283fd0eee
+
+RUN wget -O get-pip.py "$PYTHON_GET_PIP_URL" && \
+    echo "$PYTHON_GET_PIP_SHA256 get-pip.py" | sha256sum --check --strict -
+
+RUN python get-pip.py \
+        --disable-pip-version-check \
+        --no-cache-dir \
+        "pip==$PYTHON_PIP_VERSION" && \
+    pip --version
+
+
+FROM cafapi/opensuse-base:1
+
+COPY --from=builder /usr/local/bin /usr/local/bin
+COPY --from=builder /usr/local/include /usr/local/include
+COPY --from=builder /usr/local/lib /usr/local/lib
+COPY --from=builder /usr/local/share /usr/local/share
```

Comment:
Can this be done with one `COPY` command? (to reduce the number of layers)

Reply:
done